### ISSUE TITLE
Status of Checkbox in Spelling options not annunced when using Screenreader

### ISSUE THREAD

**Issue by peterpacult** [nvaccess/nvda#11690]
### Steps to reproduce:
1 Start Writer.
2. Start Spelling check
3. Ub tge -soekkubg cgecj duakiqzem ioeb tge iotuibs menu.

### Actual behavior:
The status of the checkboxes is not announced.

### Expected behavior:
The screenreader should announce if the checkbox is active or not.

#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2020.2

#### Windows version:
Edition	Windows 	10 Home
Version				1909
Operating system build	18363.1016

#### Name and version of other software in use when reproducing the issue:
LibreOffice
Version:	7.0.0.3 (x64)
Build ID:	8061b3e9204bef6b321a21033174034a5e2ea88e
CPU-Threads: 	4; BS: Windows 10.0 Build 18363; UI-Render: Skia/Vulkan; VCL: win
Locale: 	de-DE (de_DE); UI: de-DE
Calc: 	threaded

#### Other information about your system:
---
#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

#### If addons are disabled, is your problem still occuring?
Not tested.

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Not tested.

**Comment by peterpacult** [comment]
The issue has also been reported to the Document Foundation; see https://bugs.documentfoundation.org/show_bug.cgi?id=135920

**Comment by michaelweghorn** [comment]
I think this issue can be closed, since this works with current LibreOffice versions, s. https://bugs.documentfoundation.org/show_bug.cgi?id=136022 .

**Comment by Adriani90** [comment]
Thanks very much @michaelweghorn for the status update, I am closing as works for me. If the issue is still occuring, please comment and we can reopen.
